src/allencell_ml_segmenter/training/model_selection_widget.py

```diff
@@ -159,7 +159,7 @@ def __init__(
         top_grid_layout.addWidget(
             LabelWithHint(
                 label_text="Select an existing model",
-                hint="Resume training from pretrained model weights",
+                hint="Use trained model to generation segmentation prediction",
```

Comment:
Should the tooltips be complete sentances, with a period (since they are capitalized?)